financial-connections/src/main/java/com/stripe/android/financialconnections/model/FinancialConnectionsAuthorizationSession.kt

```diff
@@ -47,9 +47,9 @@ internal data class FinancialConnectionsAuthorizationSession(
     val urlQrCode: String? = null,
 
     @SerialName(value = "is_oauth")
-    val isOAuth: Boolean,
+    val isOAuth: Boolean? = false,
```

Comment:
I liked what you did with `allowSelection`: marking it `_allowSelection` and then handling the "optional" case in one place (rather than scattered throughout the code)

Reply:
yeah! Started replacing all usages and didn't seem very efficient, so moved to this approach instead : )